projects/plugins/jetpack/_inc/client/components/agencies-card/index.tsx

```diff
@@ -81,7 +86,7 @@ const AgenciesCard: FC< Props > = ( {
 					<p className="jp-agencies-card__link-button">
 						<Button>
 							<ExternalLink
-								onClick={ trackEvent( 'learn-more-click' ) }
+								onClick={ handleClick }
```

Comment:
For some reason, I am not seeing the event fire on click. I _think_ it has something to do with the the link opening in a new tab.

I tested with the click handler on the `<Button>` element and that seems to work.

Reply:
Can you make sure of that? Opening the link to jetpack.com page already generates a lot of other events, so it's easy to lose it among others: https://share.cleanshot.com/nyayGo

You can use the "Filter property" text fields in the Vigilante tool. (event `jetpack_wpa_click`).

Additionally, tracking this event depends on javascript, so make sure that you're not using any ad blocker that can disable tracking on the client (js) side.

Reply:
I can confirm I am seeing the event being fired. I am seeing `learn-more-click` not `learn-more-click-2` though like the testing instructions say. That looks correct looking at the code though

Reply:
Yep! I changed it as for the discussion in the other thread (https://github.com/Automattic/jetpack/pull/27503#discussion_r1030857784). I've updated the testing plan for documentation purposes. thanks!

Reply:
For some reason I'm unable to see the event being fired as well (filtering for `jetpack_wpa_click` and `learn-more-click` to be sure, and testing with ad blocking completely disabled).

Reply:
@coder-karen May I ask you to check in Tracks Live View if the event is not recorded on the backend side (to exclude any potential issues with Tracks Vigilante)? I'd check it myself, but I only see this comment 4 hours later, and Tracks Live View shows results up to 2 hours.

And from the analysis of other use cases of `analytics.tracks.recordEvent`, I don't see how the one presented in PR is different (e.g., `jetpack_product_card_disclaimer_click`).

Reply:
Following up here too as I followed up in Slack - just to say from my end that checking Tracks Live View didn't show anything after trying again and waiting around an hour.